gamedev/part3.md

```diff
@@ -0,0 +1,194 @@
+# Game Dev
+## Part 3: Different kinds of blocks using inheritance
+<a href="README.md#instructions">Back</a>
+
+### Intro to classes and objects
+
+A lot of our code so far has been involving the character sprite and ground sprites directly. However, there's a lot that's similar between them, and there's a lot that will be similar if we decide to create other kinds of blocks too, such as a ground block that bounces you extra high. All these different kinds of objects in the game can be considered parts of general categories of game objects, similar to how a square is a more specific kind of rectangle, which is a more specific kind of polygon, and there are other specific types of polygons such as triangles.
+
+In programming we refer to these categories as **classes**, and a more specific version of a category is referred to as a **subclass or child class** of the **parent or super class.**
```

Comment:
Introduce what a class actually is first, i.e. a collection of properties (x, y, width, height, etc.) and functions